osgpkitools/incommon_request.py

```diff
@@ -247,16 +174,13 @@ def test_incommon_connection(config, restclient):
     logger.debug('response text: ' + str(response_text))
     try:
         if response.status == 200:
-            utils.charlimit_textwrap("HTTP " + str(response.status) + " " + str(response.reason)) 
-            utils.charlimit_textwrap("Successful connection to InCommon API")
+            print("Connection successful to InCommon API") 
         else:
-            #InCommon API HTTP Error codes and messages are not consistent with documentation
-            utils.charlimit_textwrap("HTTP " + str(response.status) + " " + str(response.reason))
-            utils.charlimit_textwrap("Failed connection to InCommon API. Check your authentication credentials.")
+            # InCommon API HTTP Error codes and messages are not consistent with documentation.
+            # {Unknown user}
+            print("Connection failure to InCommon API. Check your authentication credentials.")
```

Comment:
How do you know that it is an authentication credential problem?  Why remove the response.status and response.reason?